src/swift-versions.ts

```diff
@@ -3,6 +3,8 @@ import * as core from "@actions/core";
 import { System, OS } from "./os";
 
 const VERSIONS_LIST: [string, OS[]][] = [
+  ["6.0", [OS.MacOS, OS.Ubuntu]],
+  ["6.0.0", [OS.MacOS, OS.Ubuntu]],
```

Comment:
Swift 6.0.1 is out by the way and could be added too.

(Soft bump since it would be really convenient to have this merged)

Reply:
Swift 6.0.2 is out by the way and could be added too.